rustls/src/crypto/aws_lc_rs/pq/mod.rs

```diff
@@ -1,12 +1,12 @@
-//! This crate provides a [`rustls::crypto::CryptoProvider`] that includes
-//! a hybrid[^1], post-quantum-secure[^2] key exchange algorithm --
-//! specifically [X25519MLKEM768], as well as a non-hybrid
+//! This module provides a hybrid[^1], post-quantum-secure[^2] key exchange
```

Comment:
Have realised these docs are not visible. I'll think about where to move them.

Reply:
My first thoughts are in `rustls/src/crypto/mod.rs`, or maybe a manual page ala FIPS.md?